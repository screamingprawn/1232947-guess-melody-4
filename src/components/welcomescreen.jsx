import React from 'react';
import PropTypes from 'prop-types';

const handler = () => {};

const WelcomeScreen = () => {
  return (
    <div>
      <h3 onClick={handler}>Угадай мелодию!</h3>
      <h3 onClick={handler}>У тебя есть 1 попытка.</h3>

    </div>);
};

export {WelcomeScreen};
