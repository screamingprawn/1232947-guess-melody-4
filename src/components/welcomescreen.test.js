import React from 'react';
import {WelcomeScreen} from './welcomescreen.jsx';
import renderer from 'react-test-renderer';

it(`WelcomeScreen component correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<WelcomeScreen
    mistakes={0} />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
