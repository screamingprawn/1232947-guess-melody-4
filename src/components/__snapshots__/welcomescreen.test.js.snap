// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen component correctly renders after relaunch 1`] = `
<div>
  <h3
    onClick={[Function]}
  >
    Угадай мелодию!
  </h3>
  <h3
    onClick={[Function]}
  >
    У тебя есть 
    0
     
    попыток
    .
  </h3>
</div>
`;
